static/app/types/organization.tsx

```diff
@@ -172,6 +172,15 @@ export interface TeamMember extends Member {
   teamSlug?: string;
 }
 
+/**
+ * Missing org members detected through commit authors
```

Comment:
"Missing org members" is confusing to someone who has no context of this feature. You should explain it in the comment.